addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -498,6 +498,7 @@ var FormController = BasicController.extend({
             res_model: record.model,
             title: _t("Open: ") + event.data.string,
         }).open();
+        event.stopPropagation();
```

Comment:
by convention, we do that as the first instruction of the function (see the function above)